### PR TITLE
Fix coding style issues and failing unittests 

### PR DESCRIPTION
**Pull request by stefan-caraiman**

### CODE REVIEW

**Comment by stefan-caraiman** [comment]
The `pylint: disable=wrong-import-order` is required since we are using the `restore_excepthook` method from `argus.utils` when import thirdparty libraries such as the tempest one.

**Comment by dtoncu** [comment]
LGTM

File: argus/util.py [hunk 1]
```diff
@@ -277,12 +277,11 @@ def _get_cmd(command):
...
     POWERSHELL_SCRIPT_UNRESTRICTED: _get_cmd_with_privileges("Unrestricted"),
     POWERSHELL_SCRIPT_BYPASS: _get_cmd_with_privileges("Bypass"),
-    POWERSHELL_SCRIPT_UNDEFINED: _get_cmd_with_privileges("Undefined"),
-    }
+    POWERSHELL_SCRIPT_UNDEFINED: _get_cmd_with_privileges("Undefined")}
 
 
 def get_command(command, command_type=None):
-    """Returns the command decorated acording to the command_type """
+    """Returns the command decorated acording to the command_type."""
```
Comment (ader1990) [inline argus/util.py]: according*

File: argus/unit_tests/action_manager/test_windows.py [hunk 2]
```diff
@@ -13,32 +13,38 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import requests
+# pylint: disable=no-value-for-parameter, too-many-lines, protected-access
```
Comment (ader1990) [inline argus/unit_tests/action_manager/test_windows.py:18]: remove as many pylint disables as possible

File: argus/config_generator/windows/base.py [hunk 3]
```diff
@@ -59,6 +64,7 @@ def _get_base_conf(config_name):
         base_conf = StringIO.StringIO(
             util.get_resource(config_name))
         conf = six.moves.configparser.ConfigParser()
+        # pylint: disable=deprecated-method
```
Comment (ader1990) [inline argus/config_generator/windows/base.py]: isn't there another way of handling this?

**Review by ader1990 — CHANGES_REQUESTED** [review]
there are still 3 lint disables, can you get rid of all of them (if possible)?

File: argus/config_generator/windows/base.py [hunk 4]
```diff
@@ -59,7 +64,8 @@ def _get_base_conf(config_name):
         base_conf = StringIO.StringIO(
             util.get_resource(config_name))
         conf = six.moves.configparser.ConfigParser()
-        conf.readfp(base_conf)
+        # pylint: disable=deprecated-method
+        conf.read(base_conf)
```
Comment (stefan-caraiman) [inline argus/config_generator/windows/base.py]: Read should only be called after `readfp`, please change it back to `readfp` and find another away if possible to pass the `deprecated-method` warning.

File: .pylintrc [hunk 5]
```diff
@@ -3,80 +3,106 @@
 # Specify a configuration file.
```
Comment (alexcoman) [inline .pylintrc:3]: Too many not required changes of this file.

File: argus/backends/tempest/tempest_backend.py [hunk 6]
```diff
@@ -58,6 +57,7 @@ class BaseTempestBackend(base_backend.CloudBackend):
     """
     def __init__(self, name, userdata, metadata, availability_zone):
         if userdata:
+            # pylint: disable=deprecated-method
             userdata = base64.encodestring(userdata)
```
Comment (alexcoman) [inline argus/backends/tempest/tempest_backend.py]: I think this will fail on Python 3.x.

**Review by alexcoman — CHANGES_REQUESTED** [review]

File: argus/backends/tempest/tempest_backend.py (hunk 6 above)
Comment (dtoncu) [reply argus/backends/tempest/tempest_backend.py]: With the above annotation, it will not. Any suggestion in order to avoid pylint-disable ?

**Review by ader1990 — APPROVED** [review]
lgtm

**Review by stefan-caraiman — COMMENTED** [review]
LGTM to me, but we need an Argus run for making sure nothing breaks(on both Windows 2012 and 2008), NUC1 should be up to this if needed.

**Review by alexcoman — APPROVED** [review]

**Review by alexcoman — APPROVED** [review]
LGTM;

**Pull request merged**